src/jAlert.js

```diff
@@ -679,6 +681,7 @@
             /* If there are onOpen callbacks, run them. */
             if( alert.onOpen )
             {
+                alert.onOpen.push(toggleFocusTrap);
```

Comment:
Needs to run every time. I would put this above if(alert.onOpen) (make sure it's an array first).

Reply:
Do you mean something like this?
```
            /* If there are onOpen callbacks, run them. */
            if( Array.isArray(alert.onOpen) )
            {
                alert.onOpen.push(toggleFocusTrap);
            } else {
                alert.onOpen = [alert.onOpen, toggleFocusTrap];
            }
            $.each(alert.onOpen, function(index, onOpen){
                onOpen(alert.instance);
            });
```

Reply:
I'm sorry, I just reviewed the code and it's always an array. You should just push it and loop them like this:
```
           alert.onOpen.push(toggleFocusTrap);
            $.each(alert.onOpen, function(index, onOpen){
                onOpen(alert.instance);
            });
```

please test this, but I think it's right.

Reply:
Ok thanks, just tested and pushed the changes!